src/Controller/Api/DIFController.php

```diff
@@ -467,7 +465,11 @@ public function postAction(Request $request, EntityEventDispatcher $entityEventD
      */
     public function putAction(int $id, Request $request)
     {
-        $this->handleUpdate(DIFType::class, DIF::class, $id, $request, 'PUT');
+        /** @var DIF $dif */
+        $dif = $this->handleUpdate(DIFType::class, DIF::class, $id, $request, 'PUT');
+        // Update the Dataset too because Issue Tracker Ticket.
+        $this->entityHandler->update($dif->getDataset());
```

Comment:
Before we update, should we check if the user is a DRPM, and if not, ignore/remove the Jira ticket from the dif, just in case they managed to maliciously inject it in their PUT/PATCH action?

Reply:
We would have to very deliberately check if someone was trying to change the Jira ticket. There are other options, but I feel it's not neccesary.